### PR TITLE
Deployment with env variables

### PR DESCRIPTION
**Pull request by LamaDabbeet**

### CODE REVIEW

File: pages/index.js.orig [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <a>post #2</a>
+      </Link>
+    </li>
+    <li>
+      <div>{process.env.TEST}</div>
+    </li>
+  </ul>
+
+);
+>>>>>>> 1063356d6cf9ab1a090cdbfdd7a1fdc6a15945f2
```
Comment (NourSammour) [inline pages/index.js.orig]: merge issue

File: pages/index.js.orig [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <li>
+        <Link href='/b' as='/a'>
+          <a>a</a>
+        </Link>
+      </li>
+      <li>
+        <Link href='/a' as='/b'>
+          <a>b</a>
+        </Link>
+<<<<<<< HEAD
```
Comment (NourSammour) [inline pages/index.js.orig]: merge issue

File: pages/index.js.orig [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        text-align: center;
+        transition: 100ms ease-in background;
+      }
+      .hello:hover {
+        background: #ccc;
+      }
+    `}</style>
+  </div>
+)
+=======
```
Comment (NourSammour) [inline pages/index.js.orig]: merge issue

File: next.config.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+console.log({
```
Comment (NourSammour) [inline next.config.js:2]: i prefer to keep using .env file

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Comment by LamaDabbeet** [comment]
I resolve the issues

**Review by NourSammour — APPROVED** [review]

**Pull request merged**